samples/package.json

```diff
@@ -25,9 +25,9 @@
     "updatejs": "cp build/*.js ."
   },
   "dependencies": {
-    "@google-cloud/pubsub": "^2.14.0",
```

Comment:
I think this got reverted somewhere... maybe just needs a merge from the `master` branch?

Reply:
Resolved it :)
One day I will remember which option to pick (Incoming vs Current changes) in Visual Studio Code :)

Reply:
I get confused by that terminology a lot. It seemed like it was more straightforward in the Perforce world, somehow.